### DISCUSSION TITLE
Profile read me badges (looking for feedback)

### DISCUSSION THREAD

**Discussion by ayaanvasudeva** [community/community#101759] [Programming Help]
To use them in your own readme go over <a href = "https://github.com/SuperSaiyanCoding/Profile-readme-badges">here </a>
![Screenshot 2024-01-24 11 43 53 AM](https://github.com/community/community/assets/112682624/321fae04-4178-4828-ae9f-ec10b6c0f5a1)
To contribute or add new badges: create a new issue and tell me what badge you want and add a photo

<strong>All badges are made by me copyright 2024</strong> 
Feedback is appreciated :)

**Comment by bonizario** [comment]
Yoooo, 14y and already know all that stuff? 💯 🚀 

As for the feedback, maybe reduce the size of the icons a little to increase the spacing between the name and the icon.

Anyway, awesome work!

**Reply by ayaanvasudeva** [reply]
thx! really appreciate the feedback :)
![Screenshot 2024-01-24 2 50 10 PM](https://github.com/community/community/assets/112682624/8885d08b-1b65-4cd8-9e63-078afd5da218)
I added an option so you can change the size of icon easily
![Screenshot 2024-01-24 2 52 02 PM](https://github.com/community/community/assets/112682624/68ed8309-3bd8-4f85-86f4-1008df3463c7)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @SuperSaiyanCoding ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by ghost** [comment]
Hey @SuperSaiyanCoding love your badges. This is a small suggestion add box-shadow of darker shade.This is because people viewing your github README.md in light theme would find it difficult to view the badges border radius....Just a opinion

**Reply by ghost** [reply]
14 tooo

**Reply by ghost** [reply]
![Screenshot_2024-01-24-23-50-30-477_com android chrome](https://github.com/community/community/assets/121353576/b65dc8b6-e79b-40e3-956a-0c0aa7392c05)

**Reply by ghost** [reply]
Also just another opinion on improving uniformity in you badges. As you are the creater you can see that the python logo has a 3d style and others has a flat abstract style. So I would suggest using a common style based on your liking.

**Reply by ayaanvasudeva** [reply]
Thanks for the feedback :)
I will definitely add a light & dark theme
I will probably also change the Python logo into a flat abstract style

**Reply by ayaanvasudeva** [reply]
This is how the dark theme will look like
![Screenshot_2024-01-25_11 38 53_AM-removebg-preview](https://github.com/community/community/assets/112682624/f5f84683-148b-45ce-a71b-ce754b7b8581)
Compared to the light theme
![image](https://github.com/community/community/assets/112682624/7b4c1d1d-b5c5-4b53-a9db-4ab5dcba33d3)

**Comment by Samir-Alam** [comment]
These icons really look good

**Reply by ayaanvasudeva** [reply]
Thanks!
